sdcm/ycsb_thread.py

```diff
@@ -224,7 +224,7 @@ def _run_stress(self, loader, loader_idx, cpu_idx):
         if self.stress_num > 1:
             cpu_options = '--cpuset-cpus="{cpu_idx}"'
 
-        docker = RemoteDocker(loader, "scylladb/hydra-loaders:ycsb-jdk8-20200326",
+        docker = RemoteDocker(loader, "scylladb/hydra-loaders:ycsb-jdk8-20211104",
```

Comment:
Why not take the docker name from the `docker/ycsb/image` file?

Reply:
no specific reasons.